src/_posts/languages/python/django/2000-01-01-start.md

```diff
@@ -233,6 +233,29 @@ Then change the environment variable of your application with a coma-separated l
 scalingo -a app-name env-set ALLOWED_HOSTS=app-name.osc-fr1.scalingo.io,example.com
 ```
 
+### Automatic Database Migrations
+
+If your Django application is related to a database, you can set it up to migrate your database schema automatically 
+after a deployment, thanks to the [postdeploy hook]({% post_url platform/app/postdeploy-hook %}).
```

Comment:
question: sorry I didn't noticed it at the first review. But I think that having a database is mandatory for a Django application. Hence I'm not sure the beginning of this sentence is mandatory.
```suggestion
You can configure a [postdeploy hook]({% post_url platform/app/postdeploy-hook %}) to migrate your database schema automatically after a deployment.
```